package.json

```diff
@@ -12,20 +12,23 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "test": "playwright test"
   },
   "dependencies": {
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
     "@mdx-js/react": "1.6.22",
     "@tailwindcss/forms": "0.5.3",
+    "cheerio": "1.0.0-rc.12",
```

Comment:
Is this RC version out of rc?

Reply:
Unfortunately, it's not. This version is almost two years old though.